runtime/parachains/src/ump.rs

```diff
@@ -599,51 +693,64 @@ pub(crate) mod mock_sink {
 	//! 2. All messages expected by the probe must be received by the time of dropping it. Unreceived
 	//!    messages will lead to a panic while dropping a probe.
 
-	use super::{MessageId, ParaId, UmpSink, UpwardMessage};
+	use super::{upward_message_id, MessageId, ParaId, UmpSink, UpwardMessage};
 	use frame_support::weights::Weight;
 	use std::{cell::RefCell, collections::vec_deque::VecDeque};
 
 	#[derive(Debug)]
 	struct UmpExpectation {
 		expected_origin: ParaId,
 		expected_msg: UpwardMessage,
-		mock_weight: Weight,
+		// the result returned from the `process_upward_message` for this expected message.
+		mock_result: Result<Weight, (MessageId, Weight)>,
 	}
 
 	std::thread_local! {
 		// `Some` here indicates that there is an active probe.
 		static HOOK: RefCell<Option<VecDeque<UmpExpectation>>> = RefCell::new(None);
 	}
 
+	/// Yeet an expectation into the active probe instance. Panics if there is no active probe.
```

Comment:
I actually don't know what yeet means here?

Reply:
to toss )